### ISSUE TITLE
DLPF for gyroscope

### ISSUE THREAD

**Issue by ctorney** [adafruit/Adafruit_ICM20649#17]
I think this should be using `ICM20X_B2_GYRO_CONFIG_1`
https://github.com/adafruit/Adafruit_ICM20X/blob/8a31f5345415e79703b1240c4dee28cb6dd8ca77/Adafruit_ICM20X.cpp#L587

**Comment by FL4KNinja** [comment]
Agreed. Having struggles with this issue on my Glab project too. It's strangely a non-existent problem in the Circuit-Python version of this code, but I assume an accidental copy-paste w/o remembering to edit that variable caused it. I'll likely have a fork and PR up soon, so I'll keep this thread updated
